Utilities/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     compile 'org.apache.httpcomponents:httpcore:4.4.10'
     compile 'com.mashape.unirest:unirest-java:1.4.9'
     compile 'edu.stanford.hivdb.aapcnt:hiv-aapcnt-java:2018.11-2'
-    testcompile project(':TestResources')
+    compile project(':TestResources')
```

Comment:
@arodcasanova TestResources should not be included except test environment. Can you check what are the dependencies and find a solution? Thanks.